Firestore/Example/Tests/Integration/API/FIRAggregateTests.mm

```diff
@@ -1093,4 +1093,45 @@ - (void)testPerformsAverageOnlyOnNumericFields {
                  [[NSNumber numberWithDouble:5] doubleValue]);
 }
 
+- (void)testFailWithGoodMessageIfMissingIndex {
```

Comment:
"GoodMessage" is quite strange as it did not say what is a good or bad message. Maybe change to sth like "testFailsWithLinkToConsoleMessageIfMissingIndex"

Reply:
Done.